### PR TITLE
Bioimaging Domain

### PR DESCRIPTION
**Pull request by jburel**
Add new entry for bioimaging domain

@bedroesb I wanted to use markdown table but the rendering is not great, hard to see difference between the columns
and the rows are of the same color so I opted for a long list
suggestion welcome

cc @joshmoore, @sbesson @sukunis @smza

### CODE REVIEW

**Comment by bedroesb** [comment]
@jburel Good point about the tables, I fixed this in my last commit: https://rdmkit.elixir-europe.org/markdown_cheat_sheet.html#tables (if you empty your cash it should be better)

**Comment by smza** [comment]
@jburel could you please add me as a reviewer?

**Comment by jburel** [comment]
@bedroesb could you remind me the link to the g.doc to update the list of contributors? or updating the yaml file is enough

**Comment by bedroesb** [comment]
@jburel The yaml file is enough! (even if you don't add them there they will show up, but it is nice you added some metadata)

**Review by bedroesb — COMMENTED** [review]
@jburel I wonder if we should add `bioimaging data` id to the tools discussed on this page in the [toolstable](https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668)

@korbinib and @floradanna what are your opinion on this?

**Comment by jburel** [comment]
@bedroesb That will make sense

**Comment by bedroesb** [comment]
Sure!, it is a comma separated list

**Comment by bedroesb** [comment]
The easiest is to add them to this [spreadsheet](https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668) this way it always stays up to date, or you add them to the csv and the spreadsheet, if you feel adventurous :)

**Comment by jburel** [comment]
@bedroesb I have updated the spreadsheet with the new tag and added new entries
I have also updated the CVS file and the yml file in this PR

**Comment by bedroesb** [comment]
@jburel yaml was not needed since that is overwritten automatically, but thanks anyway !

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
hmm I have to check the toolstable first since there seems to be an error

**Comment by bedroesb** [comment]
I think the csv is manually updated and at some rows the the last column got lost, I will overwrite with the one from the google spreadsheet

**Comment by bedroesb** [comment]
@jburel can you check if every tool is still correctly tagged? In general we recommend people to just add it to the google spreadsheet so they don't have to manipulate our monster csv :smiling_face_with_tear:

**Comment by jburel** [comment]
@bedroesb Thanks. It still looks good.

**Comment by jburel** [comment]
@smza Anything else needed for now? I think we will add other sections on analysis and possibly expand the metadata section in follow-up PRs

**Comment by smza** [comment]
> @smza Anything else needed for now? I think we will add other sections on analysis and possibly expand the metadata section in follow-up PRs

Hi @jburel, I have reorganised your content with additional feedback from another editor (Flora). I'll share that with you for your comments and  approval later today.

**Comment by smza** [comment]
> > @smza Anything else needed for now? I think we will add other sections on analysis and possibly expand the metadata section in follow-up PRs
> 
> Hi @jburel, I have reorganised your content with additional feedback from another editor (Flora). I'll share that with you for your comments and approval later today.

Hi @jburel here is the link to the reorganised and edited content: https://docs.google.com/document/d/1UfNW6aqzC5nzwHgjCHbEQDLgRsZOrQHFcTfkCNBbyu4/edit?usp=sharing

We've made some major changes which would be complicated to see in PR. Please have a look and see if you agree with and    let us know if you have any questions or need a discussion.

**Comment by jburel** [comment]
Thanks for the comments. I will have a look. I tried the table layout previously but it renders poorly.
How to you want to move forward? Review in the g.doc since I do not want to spend time moving between markdown and g.doc constantly

**Comment by smza** [comment]
> Thanks for the comments. I will have a look. I tried the table layout previously but it renders poorly.
> How to you want to move forward? Review in the g.doc since I do not want to spend time moving between markdown and g.doc constantly

Hi @jburel , 
I had to share the google doc because Github would not allow markdown as attachment and I couldn’t create a branch on your fork. Since there was considerable  reorganisation it would be difficult to follow if I edited your current md.   I had discussed this with Bert and we couldn’t figure out any other way.

You can either copy the content from  google doc and add it to your fork and then merge OR just add comments to google doc and I'll merge it after that.

Thanks,
Munazah

**Comment by jburel** [comment]
I have remove the changes in the CVS to avoid constant conflicts

**Comment by jburel** [comment]
@smza I have adjusted the content in this PR to match the suggestions made in the g.doc.
If any more comments, will it possible to make them on the PR itself? This will simplify things for me. Thanks.

**Comment by smza** [comment]
> @smza I have adjusted the content in this PR to match the suggestions made in the g.doc.
> If any more comments, will it possible to make them on the PR itself? This will simplify things for me. Thanks.

Thanks. I'll have a look at it. Yes from now on any comments will only be on the PR.

File: pages/your_domain/bioimaging_data.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+related_pages: 
+  your_tasks: [DMP, data organisation, data publication, existing data, transfer, licensing, metadata, storage]
+  tool_assembly: [ome, XNAT-PIC]
+---
+
+## Introduction
+
+Bioimaging specialists are acquiring an ever growing amount of data: images, associated metadata, etc. However, image data management often does not receive the attention it requires or is avoided altogether since it is considered a burdensome task. At the same time, storing images on personal computers or USB keys is no longer an option, assuming it ever was! Data volume is exponentially increasing, and not just the acquired images need storing but potentially processed images will be generated and will need to be kept alongside the original images. It is critical to proactively identify where the data will be stored, for how long, who will cover the cost of the hardware, and who will cover the cost of managing the infrastructure. All the stakeholders need to be involved in the preliminary discussions: biologists, facility managers, data analysis, IT support, etc. to ensure that the requirements are understood and met. 
+
+## What constitues bioimage data
```
Comment (bedroesb) [inline pages/your_domain/bioimaging_data.md:14]:
```suggestion
## What constitutes bioimage data
```

Can this be a typo? :)

**Review by smza — APPROVED** [review]

**Comment by smza** [comment]
@jburel I've approved and merged the PR.

**Comment by jburel** [comment]
@smza Thanks

**Pull request merged**